src/Home/index.tsx

```diff
@@ -1,28 +1,73 @@
-import React, { useState } from 'react';
-import { FontAwesome } from '@expo/vector-icons';
+import React, { useEffect, useState } from 'react';
 import {
   AddIcon,
   Box,
   Circle,
   Divider,
   HStack,
   SearchIcon,
+  Spinner,
   useToast,
   VStack,
+  Text,
+  Button,
+  Stack,
+  Center,
 } from 'native-base';
 import { ScrollView } from 'react-native';
 import { Tile } from './Tile';
 import { ModalGameJoin } from './ModalGameJoin';
 import { API_URL } from '../constants';
 import { Game } from '../interface';
 import { useNavigation } from '../navigation/useNavigation';
+import { useIsFocused } from '@react-navigation/native';
```

Comment:
isFocused means the screen is opened?

Reply:
Used to tell if the page is openend and focused or not